pkg/occlient/occlient.go

```diff
@@ -1053,37 +1054,62 @@ func (c *Client) FollowBuildLog(buildName string) error {
 }
 
 // Display DeploymentConfig log to stdout
-func (c *Client) DisplayDeploymentConfigLog(deploymentConfigName string, stdout io.Writer) error {
+func (c *Client) DisplayDeploymentConfigLog(deploymentConfigName string, followLog bool, stdout io.Writer) error {
+
+	// Set standard log options
+	deploymentLogOptions := appsv1.DeploymentLogOptions{Follow: false, Previous: true, NoWait: true}
+
+	// If the log is being followed, set it to follow / don't wait
+	if followLog {
+		// TODO: https://github.com/kubernetes/kubernetes/pull/60696
+		// Unable to set to 0, until we update our vendoring
+		// set to 1 for now.
+		tailLines := int64(1)
```

Comment:
Need your insight on this @kadel 

I'm blocked on issue https://github.com/kubernetes/kubernetes/pull/60696

The commit didn't make it into the `1.10.0` Kubernetes release freeze. It will however be in the `1.11.0` release.

At the moment, the `client-go` vendoring for OpenShift is at `1.10.2` https://github.com/openshift/client-go/blob/master/glide.yaml#L14

So we literally just have to wait until `1.11.0` of Kubernetes is released and go from there. 

The current work-around is to set it to `1`, where it will only output 1 line from the previous log, until `1.11.0` is released.